services/httpd/vhosts.nix

```diff
@@ -299,234 +256,203 @@ in (if (config.redbrick.skipVhosts) then [] else ([
+  };
+  "yfg.${tld}" = vhost {
     documentRoot = "${webtree}/f/finegael";
     user = "finegael";
     group = "associat";
-    serverAliases = [];
-  })
-  (vhost {
-    hostName = "youth2000.${tld}";
+  };
+  "youth2000.${tld}" = vhost {
     documentRoot = "${webtree}/y/youth2k";
     user = "gamessoc";
     group = "society";
-    serverAliases = [];
-  })
-  (vhost {
-    hostName = "webmail.${tld}";
+  };
+  "webmail.${tld}" = vhost {
     documentRoot = "${webtree}/vhosts/rainloop";
     user = "wwwrun";
     group = "wwwrun";
-  })
-  (vhost {
-    hostName = "bash.${tld}";
+  };
+  "bash.${tld}" = vhost {
     documentRoot = "${webtree}/y/yosarian/bash";
     user = "yosarian";
     group = "associat";
-  })
-  (vhostProxy "dcufm.${tld}" "http://136.206.16.136")
-  (vhostProxy "jakarta.${tld}" "http://136.206.15.59:8080")
-  (vhostProxy "lists.${tld}" "http://mail.internal:80")
-  (vhostProxy "macspayn.${tld}" "http://136.206.15.25:3007")
-  (vhostProxy "portaldev.${tld}" "http://136.206.15.61:9080")
-  (vhostProxy "radio.${tld}" "http://radio.redbrick.dcu.ie:8000")
-  (vhostProxy "riainccc.${tld}" "http://http://136.206.15.25:3000")
-  (vhostProxy "tomcat.dregin.${tld}" "http://136.206.15.14:20002")
-  (vhostProxy "webchat.${tld}" "http://127.0.0.1:16667")
-  (vhostProxy "werdztomcat.${tld}" "http://136.206.15.14:20001")
-  (vhostRedirect "www.${tld}" "https://${tld}")
-  (vhostRedirect "admin.${tld}" "https://blog.redbrick.dcu.ie")
-  (vhostRedirect "admins.${tld}" "https://blog.redbrick.dcu.ie")
-  (vhostRedirect "ajaxterm.${tld}" "https://term.redbrick.dcu.ie")
-  (vhostRedirect "anyterm.${tld}" "https://term.redbrick.dcu.ie")
-  (vhostRedirect "dconcannon.${tld}" "https://www.redbrick.dcu.ie/~shimoda")
-  (vhostRedirect "dermot.${tld}" "https://www.redbrick.dcu.ie/~homer")
-  (vhostRedirect "devnull.${tld}" "https://www.redbrick.dcu.ie/~colmmacc")
-  (vhostRedirect "devrandom.${tld}" "https://www.redbrick.dcu.ie/~marvin")
-  (vhostRedirect "events.${tld}" "https://redbrick.dcu.ie/events")
-  (vhostRedirect "fosdem.${tld}" "https://redbrickdcu.typeform.com/to/ZwETj0")
-  (vhostRedirect "github.${tld}" "https://github.com/redbrick")
-  (vhostRedirect "help.${tld}" "https://wiki.redbrick.dcu.ie/mw/Helpdesk")
-  (vhostRedirect "helpdesk.${tld}" "https://wiki.redbrick.dcu.ie/mw/Helpdesk")
-  (vhostRedirect "helpdeskexam.${tld}" "https://md.redbrick.dcu.ie/s/SJzip7F9X#")
-  (vhostRedirect "hoodies.${tld}" "https://redbrickdcu.typeform.com/to/Q4uIzR")
-  (vhostRedirect "parlour.${tld}" "http://www.songsfromtheparlour.com")
-  (vhostRedirect "radio.theinternets.be" "http://radio.redbrick.dcu.ie")
-  (vhostRedirect "techweek.${tld}" "https://techweek.dcu.ie")
-  (vhostRedirect "tickets.${tld}" "https://dcusu.ticketsolve.com/shows/873599383/events/128190598")
-  (vhostRedirect "ubuntu.${tld}" "https://wiki.redbrick.dcu.ie/mw/RedBrick_Ubuntu")
-] ++ userVhosts))
+  };
+  "dcufm.${tld}" = vhostProxy "http://136.206.15.74";
```

Comment:
we seem to be missing all the metrics services. did you rebase of master?

Reply:
Not yet, I won't be doing that until I get mail ready to merge. I'll cover those then

Reply:
Yeah bare in mind this merges into the `mail` branch